apps/src/templates/teacherDashboard/LoginTypeCard.jsx

```diff
@@ -94,14 +96,6 @@ class LoginTypeCard extends Component {
             {description}
           </div>
         </div>
-        <Button
-          className="uitest-button"
```

Comment:
This makes me think that there is a UI test that will fail because it's looking for this button, but it won't be there.  If that's the case, we'll have to update the UI test to match the change. Let's see if Circle passes...